internal/eva/create_organizaion_unit.go

```diff
@@ -0,0 +1,63 @@
+package eva
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+type CreateOrganizationUnitRequest struct {
+	Name                string `json:"Name"`
+	PhoneNumber         string `json:"PhoneNumber"`
+	EmailAddress        string `json:"EmailAddress"`
+	ParentID            int64  `json:"ParentID"`
+	CurrencyID          string `json:"CurrencyID"`
+	BackendID           string `json:"BackendID"`
+	CostPriceCurrencyID string `json:"CostPriceCurrencyID"`
+}
+
+type createOrganizationUnitRequest struct {
+	ToCreate CreateOrganizationUnitRequest `json:"ToCreate"`
+}
+
+type CreateOrganizationUnitResponse struct {
+	ID int64 `json:"ID"`
+}
+
+func (c *Client) CreateOrganizationUnit(ctx context.Context, req CreateOrganizationUnitRequest) (*CreateOrganizationUnitResponse, error) {
+	const (
```

Comment:
It makes sense to make this a constant but I'm not sure about it being inside the method definition itself. Maybe extract this into the higher scope and rename to `createOrganizationUnitEndpoint` or `createOrganizationUnitPath`

Reply:
Yeah, Initially I wanted a single file per endpoint where the `path` would be the same common thing. So each endpoint would essentially be a copy/paste of the other file where the only thing that would need to be updated would be the structs. However having the same constant defined outside of the method scope is not allowed in go, so the only way to reuse the same `path` const was to move the constant to the method scope (which I also dont like). 

I think you suggestion of migrating all methods to `organization_units.go` and create a const at file level with unique names is most likely the way to go.